### PR TITLE
Test Module for testing code censor.

### PR DESCRIPTION
**Pull request by gauravgoyal**

### CODE REVIEW

File: test_modules/test_module_2/test_module_2.info.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: 'NCM Search'
+description: 'Contains Search related configuration.'
+type: module
+core: 8.x
+dependencies:
+  - blocktabs:blocktabs
```
Comment (gupta-ashutosh5) [inline test_modules/test_module_2/test_module_2.info.yml:6]: **Add package information in custom module info file**
Generally project specific name as package should be added

 **References:**
 [Custom Module Development](https://www.drupal.org/docs/8/creating-custom-modules/let-drupal-8-know-about-your-module-with-an-infoyml-file)

File: test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function addFieldValues(ItemInterface $item) {
+    $object = $item->getOriginalObject()->getValue();
+    if ($object->getType() == 'job') {
+      $address = $object->get('field_job_address')->getValue();
+      $country_code = $address[0]['country_code'];
+      $countries = \Drupal::service("address.country_repository")->getList();
```
Comment (saitanay) [inline test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php]: **Avoid directly calling Drupal::service(), instead inject the service.**
Dependency injection is the preferred method for accessing and using services in Drupal 8 and should be used whenever possible.

 **References:**
 [Services and Dependency Injection](https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8#using-dependency-injection)

File: test_modules/test_module_1/test_module_1.module [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (saitanay) [inline test_modules/test_module_1/test_module_1.module:1]: **Looks like module hook_help is missing**
Always implement hook_help in newly created custom module

File: test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php (hunk 2 above)
Comment (saitanay) [inline test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php]: **Avoid directly calling Drupal::service(), instead inject the service.**
Dependency injection is the preferred method for accessing and using services in Drupal 8 and should be used whenever possible.

 **References:**
 [Services and Dependency Injection](https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8#using-dependency-injection)

File: test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      $country_name = $countries[$country_code];
+      $fields = $item->getFields(FALSE);
+      $fields = $this->getFieldsHelper()->filterForPropertyPath($fields, NULL, 'ncm_country_code_synonym');
+      foreach ($fields as $field) {
+        $synonyms = [];
+        $query = \Drupal::entityQuery('search_api_synonym')
```
Comment (saitanay) [inline test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php]: **Avoid directly calling Drupal::entityQuery(), instead inject the entity.query service.**
Dependency injection is the preferred method for accessing and using services in Drupal 8 and should be used whenever possible.

 **References:**
 [Services and Dependency Injection](https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8#using-dependency-injection)

File: test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php (hunk 2 above)
Comment (saitanay) [inline test_modules/test_module_2/src/Plugin/search_api/processor/NcmCountryCodeSynonym.php:53]: **Avoid directly calling Drupal::service(), instead inject the service.**
Dependency injection is the preferred method for accessing and using services in Drupal 8 and should be used whenever possible.

 **References:**
 [Services and Dependency Injection](https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8#using-dependency-injection)

**Pull request closed without merging**